package.json

```diff
@@ -20,6 +20,7 @@
     "eslint-loader": "4.0.2",
     "eslint-plugin-import": "2.22.0",
     "file-loader": "6.0.0",
+    "gulp-sass": "^4.1.0",
```

Comment:
This needs to be uninstalled. 
```suggestion
```

Reply:
We're reading through #98, and we saw that Nathan has 14.15.4 of node. The issue ticket says that Honey Rae had to downgrade to 14. But it looks like Nathan is already on 14? Am I interpreting this wrong?

Reply:
You're not. We're continuing this conversation in the DIscussion Ticket [here](https://github.com/nss-evening-cohort-14/evening-client-side/discussions/194)